conan/internal/default_settings.py

```diff
@@ -53,23 +53,23 @@
                     "18.0", "18.1", "18.2", "18.3", "18.4"]
         sdk: ["appletvos", "appletvsimulator"]
         sdk_version: [null, "11.3", "11.4", "12.0", "12.1", "12.2", "12.4",
-                        "13.0", "13.1", "13.2", "13.4", "14.0", "14.2", "14.3", "14.5", "15.0",
-                        "15.2", "15.4", "16.0", "16.1", "16.4", "17.0", "17.1", "17.2", "17.4", "17.5",
-                        "18.0", "18.1", "18.2", "18.3", "18.4"]
+                        "13.0", "13.2", "13.3", "13.4", "14.0", "14.2", "14.3", "14.4", "14.5", "15.0",
+                        "15.2", "15.4", "15.5", "16.0", "16.1", "16.4", "17.0", "17.1", "17.2", "17.4", "17.5",
+                        "18.0", "18.1", "18.2", "18.4"]
     visionOS:
-        version: ["1.0", "1.1", "1.2", "2.0", "2.1", "2.2", "2.3", "2.4"]
+        version: ["1.0", "1.1", "1.2", "1.3", "2.0", "2.1", "2.2", "2.3", "2.4"]
```

Comment:
There is a 1.3 missing visionOS release https://developer.apple.com/documentation/visionos-release-notes/visionos-1_3-release-notes